pyvista/core/filters/poly_data.py

```diff
@@ -16,6 +16,7 @@
 from pyvista.core._typing_core import NumpyArray
 from pyvista.core.errors import MissingDataError
 from pyvista.core.errors import NotAllTrianglesError
+from pyvista.core.errors import PyVistaDeprecationWarning
```

Comment:
Need to import `assert_empty_kwargs`
```suggestion
from pyvista.core.errors import PyVistaDeprecationWarning
from pyvista.core.utilities.misc import assert_empty_kwargs
```

Reply:
not needed, I removed `kwargs`